### ISSUE TITLE
VoxelTerrain SDF unusably slow

### ISSUE THREAD

**Issue by tinmanjuggernaut** [Zylann/godot_voxel#117]
VoxelTerrain SDF is unusably slow due to this error message being spammed in the console.

```
At: modules\voxel\meshers\blocky\voxel_mesher_blocky.cpp:377
ERROR: VoxelMesherBlocky::build: Condition "voxels.get_channel_compression(channel) != VoxelBuffer::COMPRESSION_NONE" is true.
```

Commenting out that line returns it to its normal speed:
```
ERR_FAIL_COND(voxels.get_channel_compression(channel) != VoxelBuffer::COMPRESSION_NONE);
```

**Comment by Zylann** [comment]
I made a fix in 8c16424af1335cece659c95cd773106b267f31aa

**Comment by tinmanjuggernaut** [comment]
Thank you, works great.
